test/cpp/grpclb/grpclb_test.cc

```diff
@@ -0,0 +1,688 @@
+/*
```

Comment:
Yes. I needed a way to deal with protobufs without going crazy. The current C protobuf support (via nanopb) doesn't have the code to generate requests, only to parse them.